Auth/Auth.Platform.iOS/AuthService.cs

```diff
@@ -7,6 +7,7 @@
 using AeroGear.Mobile.Core.Configuration;
 using AeroGear.Mobile.Core.Storage;
 using AeroGear.Mobile.Auth;
+using AeroGear.Mobile.Core.Utils;
```

Comment:
not sure if this is required given there are no other changes in this file.

Reply:
Good catch. It is a leftover from previous code.